index.html

```diff
@@ -29,61 +29,76 @@ <h1>Some Text here</h1>
     </header> 
      
        <article>
+        <h2 id="intro">Intro</h2>
+            <p>Tools used in the tech to accomplish a goal are just as essential as the code itself from file sharing, communication, document processing and workflow if we didn’t have these tools, we as developers wouldn't be able to do our jobs. 
+            <br>These tools streamline the effort of coding and make collaboration among teams much more efficient.
+            <br>As developers we need to communicate with those on our team and cross functional teams as well.
+            <br>Tools in tech makes this easier and allows us to make sure the technology we are building works and is meeting the standards of all those involved. </p>
+           
         <h2 id="eclipse">Eclipse</h2>
-        <p></p>
-         
+            <p>Eclipse is an IDE developed and mainly based in Java, but can be used to develop in other popular languages like c/c++, Javascript, PHP and Ruby. 
+            <br>It was developed by the Eclipse Foundation released in 2001. 
+            <br> At its base Eclipse supports an incremental compiler, only recompiling portions of the program that have been modified instead of recompiling the whole thing. It also includes a model of the program contents, allowing for detailed information to be viewed. Eclipse also boasts several other tools to assist in charting, reporting, and testing.
+            <br>Eclipse is open source, thanks to which has a large marketplace of extensions and plugins to meet your needs whether it be a code checker or simple skin for you editor shell. It supports Windows, Linux, and the Mac OS X.</p>
+            <h4>Basic Info:</h4>
+            <p><br>Pricing: Free 
+            <br>Type: Integrated Development Environment
+            <br>Sectors: Software Development, IT, Web Development
+            <br>Mobile: No </p>
+          
         <h2 id="slack">Slack</h2>
-        <p>Slack is a cloud- based proprietary instant messaging platform that was first released in august 2013 it is the fastest growing SaaS company of all time went from being valued at 0 to 4 billion in just 4 years. 
-        <p>Slack offers its users a free option and two paid plans(Standard at $8, Plus at $15)</p>  
-        <p>Slack has managed to capture interest in all fields of industry a numerous amount of companies including Airbnb, Lyft, Pinterest, and Dropbox use slack in their tech stack by removing the hassle of emails and adding a bit more 
-        the interesting thing about slack are the search synchronization and simple file sharing </p> 
-        <a href="http://slack.com">Slack.com</a>
-           <figure>

```

Comment:
I want to keep this portion to see how the page looks with this tagging format, and the file names